src/ansible_creator/subcommands/init.py

```diff
@@ -117,11 +115,8 @@ def unique_name_in_devfile(self) -> str:
         Returns:
             Unique name entry.
         """
-        final_name: str
-        if self._project == "collection":
-            final_name = f"{self._namespace}.{self._collection_name}"
-        if self._project == "ansible-project":
-            final_name = f"{self._scm_org}.{self._scm_project}"
+        final_name: str = ""
+        final_name = f"{self._namespace}.{self._collection_name}"
```

Comment:
Oh whoops this isn't needed
```suggestion
        final_name = f"{self._namespace}.{self._collection_name}"
```